packages/kit/src/views/Market/MarketHomeV2/components/MarketTokenListNetworkSelector/MarketNetworkFilterMobile.tsx

```
@@ -4,7 +4,7 @@ import { ScrollView, XStack } from '@onekeyhq/components';
 import type { IListViewProps, IPopoverProps } from '@onekeyhq/components';
 import type { IServerNetwork } from '@onekeyhq/shared/types';
 
-import { GradientMask } from './GradientMask';
+import { GradientMask } from '@onekeyhq/components';
```

ESLint

[error] '@onekeyhq/components' imported multiple times.

(import/no-duplicates)